styles/components/_playersBoard.scss

```diff
@@ -0,0 +1,26 @@
+.board-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+.players__data {
+  display: flex;
+  margin: 50px;
+  &__avatar {
+    width: 300px;
+  }
+  &__name {
+    font-size: 50px;
+  }
+}
+.players__herd {
```

Comment:
herd could be incorporated in one players section of styles:
`.players{
  &__data{
    ...
    }
   &__herd{
     ...
     }